### PR TITLE
Start of the PrestaShop project newsletter

### PR DESCRIPTION
**Pull request by kpodemski**
Join the PrestaShop Project newsletter! 🚀

### CODE REVIEW

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+At PrestaShop, we are always looking for ways to strengthen our relationship with the community. In line with this, we are thrilled to announce the launch of the PrestaShop Project Newsletter, a new initiative designed to bring you technical insights, exclusive updates, and educational content like never before.
+
+The launch of this newsletter is happening now, and the timing is no coincidence! With the [PrestaShop Developer Conference](https://events.prestashop.com/prestashop-developer-conference/) just around the corner on November 6, 2024, we want to highlight our commitment to the technical community. Whether you are a developer, a contributor, or simply interested in the future of PrestaShop, this newsletter is an opportunity to stay ahead of the curve.
+
+## Why Subscribe to the PrestaShop Project Newsletter?
+
+The PrestaShop Project Newsletter will keep you updated on all technical things in the PrestaShop ecosystem. Here's what you can expect when you sign up:
+
+**1. Exclusive Technical Insights and Updates**
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]:
```suggestion
**1. Exclusive technical insights and updates**
```

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Stay informed about the latest advancements, features, and technical improvements in the PrestaShop software. Our newsletter will be your go-to source for deep dives into updates, best practices, and behind-the-scenes information that will help you make the most of the platform.
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]:
```suggestion
Stay informed about the latest advancements, features, and technical improvements in PrestaShop software. Our newsletter will be your go-to source for deep dives into updates, best practices, and behind-the-scenes information that will help you make the most of the platform.
```

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+**2. Regular Software Releases and New Features**
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]:
```suggestion
**2. Regular software releases and new features**
```

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Learn about the details of new versions, releases, and enhancements that can improve your online store's performance, security, and functionality.
+
+**3. Community Projects, Events and more**
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]:
```suggestion
**3. Community projects, events and more**
```

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  - 2 vouchers for **25% off training sessions** at PrestaShop Academy.
+  </div>
+</div>
+
+<a href="javascript:void(0);" onclick="toggleTerms()">Show terms and conditions</a>
+
+<div id="terms" style="display: none;">
+This offer is organized by PrestaShop SA.
+
+Participants who have registered with a valid email address for the PrestaShop Project Newsletter between October 8th and November 5th are eligible for the contest.
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]: The starting date is October 8, when the blog post is published on Oct 21.

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+The PrestaShop team will randomly select 6 winners and announce them during the PrestaShop Developer Conference on November 6, 2024. The draws are not cumulative. In the days following the draw, the winners will also be notified using the email address they provided when registering for the newsletter.
+
+The prizes are awarded in the form of:
+- 2 vouchers for 100 EUR valid on the PrestaShop Marketplace, usable until DD/MM/YYYY
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md]: Dates in this paragraph are DD/MM/YYYY

Comment (coffrep) [reply content/news/posts/2024/introducing-prestashop-project-newsletter.md]: Just to let you know...

File: themes/build/layouts/partials/author/post-footer.html [hunk 7]
```diff
@@ -40,3 +40,64 @@ <h5>About the author</h5>
     </div>
   </div>
 </div>
+
+
+<div class="newsletter-section">
+  <div class="newsletter-text">
+    <p>Want to stay up to date with the PrestaShop project and the technical development of PrestaShop?</p>
```
Comment (coffrep) [inline themes/build/layouts/partials/author/post-footer.html]:
```suggestion
    <p>Want to stay up-to-date with the PrestaShop project and the technical development of PrestaShop?</p>
```

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+Discover the latest community-driven projects, get details on upcoming events, and enjoy exclusive content that gives you a broader perspective on what’s happening in the PrestaShop ecosystem. 
+
+<style>
+  #prizes-for-our-newsletter-subscribers { margin-top: 0; border-bottom: 1px solid #000; }
+</style>
+<div class="newsletter-prizes-announcement" style="padding: 10px; background:  #a2e2ef;">
+  <div style="padding: 30px 20px; background: #a2e2ef; border: 10px solid white;">
+  <h2 id="prizes-for-our-newsletter-subscribers">Prizes for Our Newsletter Subscribers! 🏆</h2> 
+
+  To celebrate the launch, PrestaShop SA is offering 6 exciting prizes* for those who subscribe before the Developer Conference. The prize winners will be announced during the PrestaShop Developer Conference on November 6, 2024. The prizes include:
```
Comment (coffrep) [inline content/news/posts/2024/introducing-prestashop-project-newsletter.md:37]: In this paragraph I see there is an asterisk * But I do not see where this asterisk takes you, do let me know if I'm missing something @kpodemski thanks a lot!

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi @kpodemski many thanks for asking for the review. I added a few changes and also a few comments, great if you can let me know of your thoughts regarding the comments on the blog post. I'm "requesting changes" so you can let me know about the comments, don't hesitate on pinging me on the topic. Great if you can ask for a second review so I can read once again. Thanks a lot!

File: content/news/posts/2024/introducing-prestashop-project-newsletter.md (hunk 8 above)
Comment (kpodemski) [reply content/news/posts/2024/introducing-prestashop-project-newsletter.md:37]: I added asterisk to "Show terms and conditions"

**Review by coffrep — APPROVED** [review]
Hi @kpodemski I'm approving, good for publishing! Thanks a lot.

**Pull request merged**